instrumentation/apache-shenyu/apache-shenyu-2.4/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.apache.shenyu")
+    module.set("shenyu-web")
+    versions.set("[2.4.0,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  compileOnly("org.apache.shenyu:shenyu-web:2.4.0")
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+
+  testInstrumentation(project(":instrumentation:netty:netty-4.1:javaagent"))
+  testLibrary("org.springframework.boot:spring-boot-starter-test:2.0.0.RELEASE")
+
+  testImplementation("org.springframework.boot:spring-boot-starter-webflux:2.2.2.RELEASE") {
+    exclude("org.codehaus.groovy", "groovy")
+    exclude("org.springframework.boot", "spring-boot-actuator")
+  }
+  // based on apache shenyu 2.4.0 official example
+  testImplementation("org.apache.shenyu:shenyu-spring-boot-starter-gateway:2.4.0") {
+    exclude("org.codehaus.groovy", "groovy")
+  }
+
+  latestDepTestLibrary("org.springframework.boot:spring-boot-starter-test:2.7.+")
+  latestDepTestLibrary("org.apache.shenyu:shenyu-spring-boot-starter-gateway:2.+") {
```

Comment:
there is no `3.x` version, why do you need to limit latest to `2.x`? Also you have the same artifact in `testImplementation` and `latestDepTestLibrary` you could use `testLibrary` instead.

Reply:
done